src/main/startSandbox.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+import unpackByOutpoint from './unpackByOutpoint';
+
+// Polyfills and `lbry-redux`
+global.fetch = require('node-fetch');
+
+global.window = global;
+// eslint-disable-next-line import/no-commonjs,global-require
+const { Lbry } = require('lbry-redux');
```

Comment:
Does this need to be here? Or could we use `import` and move it to the top, then remove the eslint disable line.

Reply:
Try it with `import`, doesn't work.  See my other comment.